src/Jabbr-Core.Data/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
```

Comment:
This will need to be cleaned up (along with a bunch of the other files) with the changed @ashanhol will make converting it back to a class library - as of now its a website